services/ui-src/src/routes/Dashboard/DashboardPage.test.tsx

```diff
@@ -88,7 +90,7 @@ describe("Test Dashboard view (with reports, desktop view)", () => {
     jest.clearAllMocks();
   });
 
-  test("Check that Dashboard view renders", () => {
+  test("DAHN Check that Dashboard view renders", () => {
```

Comment:
typo?

Reply:
Nope -- DAHN needed to make sure that test was working. I subcontract things to him sometimes